CONTRIBUTING.md

```diff
@@ -610,6 +610,32 @@ Then put this:
 export NODE_OPTIONS=--no-experimental-fetch
 ```
 
+If Following type of error comes while running dev server(i.e using above commands):
+
+```bash
+Error: ENOSPC: System limit for number of file watchers reached
+```
+The error is thrown because the number of files monitored by the system has reached the limit.
+The error can be fixed by increasing the amount of inotify watchers.
+
+Do the following to Modify the number of system monitoring files :
+
+
+The current value of max watches can be checked by below command.
+```bash
+cat /proc/sys/fs/inotify/max_user_watches
+```
+Edit the file /etc/sysctl.conf to increase this value.
+The value needs to be decided based on the system memory.
+
+Open the file in editor and add a line at the bottom specifying the max watches values.
+```bash
+fs.inotify.max_user_watches=524288
+```
+save the file  and exit
+```bash
```

Comment:
Can you add a line before this command explaining what it does?

Reply:
done